.github/workflows/build_upload.yml

```diff
@@ -0,0 +1,49 @@
+name: Build & Upload atomfeed-console
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - "**.md"
+
+env:
+  ORG_NAME: Bahmni
```

Comment:
ICT4H

Reply:
This ORG_NAME is being resolved by the curl url. Since we are triggering a repo in Bahmni org